package.json

```
@@ -114,7 +113,7 @@
         "src/**/*.ts"
       ],
       "output": [
-        "lib/**"
+        "lib/cjs/*"
```

Explicitly point to dir so `wireit` creates the proper cache else every rerun would replace the file leading to a possible bricked state.

---

Don't you want `lib/cjs/**` here, to include subdirectories as well?

---

Ops, that is correct, thank you for noticing! 
```suggestion
        "lib/cjs/**"
```